docs/docs/quickstarts/login/nextjs.md

```diff
@@ -95,6 +95,7 @@ Create a file `.env` in the root of the project and add the following keys to it
 ```
 NEXTAUTH_URL=http://localhost:3000
 ZITADEL_CLIENT_ID=[yourClientId]
+ZITADEL_ISSUER=[your-instance].zitadel.cloud
```

Comment:
i would make clear that this is your domain.
It dosnt need to be .zitadel.cloud
and your instance is not completly correct: its instancename-randomsting.zitadel.cloud